Sources/wkhtmltopdf/Document+Generate.swift

```diff
@@ -1,48 +1,52 @@
 import Foundation
-import Core
+import Service
 
 #if os(Linux) && !swift(>=3.1)
```

Comment:
Since we are now targeting Swift 4.2, there is no need for this conditional typealias any more.